manifest.json

```diff
@@ -6,17 +6,17 @@
 		"en": "Mopidy - is an extensible music server written in Python.",
 		"fr": "Mopidy - est un serveur de musique extensible écrit en Python"
 	},
-	"version": "1.0.0",
+	"version": "1.0.2",
 	"url": "https://www.mopidy.com/",
 	"license": "free",
 	"maintainer": {
```

Comment:
You may want to put you name as maintainer and creat a new entry with previous maintainer